gradle/libs.versions.toml

```diff
@@ -4,13 +4,13 @@ firebaseCommonKtx = "20.4.2"
 firebaseFirestore = "1.8.1"
 firebase-app-distribution = "4.0.1"
 firebase = "1.11.1"
-kotlin = "1.9.20"
+kotlin = "1.9.22"
 kotlin-json = "1.6.1"
-compose = "1.5.4"
```

Comment:
The app is crashing when I try to run it. Could it be because of the version change?
                                                                                                    java.lang.NoClassDefFoundError: Failed resolution of: Landroidx/compose/material3/SearchBarKt;

Reply:
Hey I decreased compose-material3 => "1.1.2", what is supposed to be use instead of SearchBar from 1.2.0